docs/usage/registry-cache/upstream-credentials.md

```diff
@@ -73,6 +73,8 @@ To rotate registry credentials perform the following steps:
 1. Make sure that the old Secret is no longer referenced by any Shoot cluster. Finally, delete the Secret containing the old credentials (e.g., `ro-docker-secret-v1`).
 1. Delete the corresponding old credentials from the cloud provider account.
 
+> **Note**: Do not delete old Secret before the Shoot is updated with the new Secret and Shoot reconciliation is successful.
```

Comment:
I don't recall what was our idea with this note. Few lines above there is ` Make sure that the old Secret is no longer referenced by any Shoot cluster` which seems to be the same as this note.
Maybe we talked to add a general note (not only in the context of the secret rotation) that a Secret referenced by the Shoot should not be deleted? If that's the case, the let's move this note of out the section for the registry credentials.